presto-spi/src/main/java/com/facebook/presto/spi/statistics/DoubleRange.java

```diff
@@ -78,6 +80,7 @@ public int hashCode()
         return Objects.hash(min, max);
     }
 
+    @JsonValue
```

Comment:
Let's annotate `getMin` and `getMax` with the `@JsonProperty` to keep this class Json parsable